test/e2e/nodeportlocal_test.go

```diff
@@ -365,8 +365,8 @@ func NPLTestMultiplePods(t *testing.T, data *TestData) {
 	clientNode := nodeName(0)
 	serverNode := nodeName(1)
 	if len(clusterInfo.windowsNodes) != 0 {
-		serverNode = workerNodeName(clusterInfo.windowsNodes[0])
```

Comment:
Why switch the Node index for serve and client roles? Maybe add some comments if it is necessary.